client/ioc-compose.yml

```diff
@@ -1,13 +1,12 @@
-
 services:
   ioc1:
-    build: ../client
+    build: .
```

Comment:
I'll need to check if this breaks https://github.com/ChannelFinder/recsync/blob/master/server/tests/docker/test-bash-ioc.yml